Sources/TvOSPinKeyboardViewController.swift

```diff
@@ -206,6 +206,7 @@ open class TvOSPinKeyboardViewController: UIViewController {
     
     private func setUpDeleteButton() {
         deleteButton = numpadButton(withTitle: deleteButtonTitle)
+        deleteButton.accessibilityIdentifier = "PinKeyboard.Button.Delete"
```

Comment:
What do you think about also adding an `accessibilityLabel` to the delete button?. By default its title is "←" which is not very friendly to Voice Over.
We could improve that by adding a new property `deleteButtonAccessibilityLabel` to TvOSPinKeyboard to allow the client Apps to set a better accessibility label for it.